test/tests/misc/test_harness_tester/tests

```diff
@@ -1,4 +1,8 @@
 [Tests]
+  [./test_runcommand]
+    type = 'RunCommand'
+    command = 'echo Hello World'
+  [../]
```

Comment:
No because some of them cause the TestHarness to fail... correctly (parse errors and cyclic dependencies). If we want to test the TestHarness, we'll need a different recipe for that.